Resources/bin/build_demo.sh

```diff
@@ -101,7 +101,6 @@ cd $TARGET/twig/extensions && rm -rf README doc phpunit.xml* test
 
 # final cleanup
 find $TARGET -name .git | xargs rm -rf -
-find $TARGET -name .gitignore | xargs rm -rf -
```

Comment:
this could still be valid, if you ensure that you preserve the root one (as done in the initial cleanup)